frontend/src/components/auth/Auth.vue

```diff
@@ -1,5 +1,6 @@
 <template>
    <v-container class="mx-auto px-4">
+      <h1 class="text-center">Todo App</h1> 
```

Comment:
The header should be "Awesome Todo Application"

Reply:
Sure, I will fix it.